operator/v1/types_ingress.go

```diff
@@ -716,8 +720,61 @@ type AWSNetworkLoadBalancerParameters struct {
 	// +optional
 	// +openshift:enable:FeatureGate=IngressControllerLBSubnetsAWS
 	Subnets *AWSSubnets `json:"subnets,omitempty"`
+
+	// eipAllocations assign Elastic IP addresses to the Network Load Balancer.
+	//
+	// The following restrictions apply:
+	//
+	// eipAllocations cannot be used with "Internal" but "External" scope.
+	//
+	// The number of Allocation IDs must match the number of subnets that are used for the load balancer.
+	//
+	// An EIP can be allocated to only a single IngressController.
+	//
+	// Each entry in the eipAllocations list must be unique.
+	//
+	// A maximum of 10 IP allocations are permitted.
+	//
+	// See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html for general
+	// information about configuration, characteristics, and limitations of Elastic IP addresses.
+	// +openshift:enable:FeatureGate=SetEIPForNLBIngressController
+	// +optional
+	// +listType=atomic
+	// +kubebuilder:validation:XValidation:rule=`self.all(x, self.exists_one(y, x == y))`,message="eipAllocations cannot contain duplicates"
+	// +kubebuilder:validation:MaxItems=10
+	EIPAllocations []EIPAllocation `json:"eipAllocations"`
 }
 
+// EIPAllocation is a reference to an EIP that can be allocated to an ELB in the AWS environment.
```

Comment:
This type doesn't outright mention that it's an ID. EIPs have name and ID, so it's probably best to be clear about that. 

```suggestion
// EIPAllocation is an ID for an Elastic IP (EIP) address that can be allocated to an ELB in the AWS environment.
```

Reply:
ok done